### PR TITLE
Add `_.clamp` and tests.

### PR DESCRIPTION
**Pull request by Xotic750**
https://github.com/lodash/lodash/issues/1549

### CODE REVIEW

File: lodash.js [hunk 1]
```diff
@@ -11005,6 +11005,29 @@
...
+     * @example
+     *
+     * _.clamp(-10, -5, 5);
+     * // => -5
+     *
+     * _.clamp(10, -5, 5);
+     * // => 5
+     */
+    function clamp(number, min, max) {
+      return Math.min(Math.max(number, min), max);
```
Comment (jdalton) [inline lodash.js]: We have `nativeMin` and `nativeMax` vars for use.

**Comment by Xotic750** [comment]
Ok, didn't realise.

**Comment by Xotic750** [comment]
Is that enough tests or do you require any more to define behaviour?

**Comment by jdalton** [comment]
You could add a test for floating-point numbers.

**Comment by Xotic750** [comment]
Ok, will do.

**Comment by jdalton** [comment]
Cool, thanks! I'll turn around and reuse this method in lots of places in the source.

**Comment by jrf0110** [comment]
Whatever happened to this? I don't see clamp mentioned in the docs.

**Comment by jdalton** [comment]
@jrf0110 It's in the edge master branch. We haven't version bumped yet. v4 is due out in Jan.

**Pull request merged**